src/main/kotlin/me/spradling/gift/core/api/Application.kt

```diff
@@ -12,6 +13,8 @@ class Application @Inject constructor(val vertx: Vertx, val restVerticle: RestVe
     fun main(args: Array<String>) {
       Json.mapper.disable(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES)
       Json.prettyMapper.disable(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES)
+      Json.mapper.setSerializationInclusion(JsonInclude.Include.NON_NULL)
```

Comment:
nit: i'd just create a private function here like `configureJsonMappers` or something for readability

Reply:
Done.